provider/ec2/instancetypes.go

```diff
@@ -0,0 +1,328 @@
+// Copyright 2016 Canonical Ltd.
```

Comment:
file was renamed from instance_information.go and some methods from the old internal/ec2instancetypes package copied across.
supportedInstanceTypes() and instancetypeCosts() are new